it/v1/endpoints/ListBenefitsControllerISpec.scala

```diff
@@ -197,32 +225,34 @@ class ListBenefitsControllerISpec extends IntegrationBaseSpec {
         input.foreach(args => (validationErrorTest _).tupled(args))
       }
 
-      "des service error" when {
-        def serviceErrorTest(desStatus: Int, desCode: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
-          s"des returns an $desCode error and status $desStatus" in new Test {
-
+      "downstream service error" when {
+        def serviceErrorTest(downstreamStatus: Int, downstreamCode: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
+          trait HasTest { _:Test =>
             override def setupStubs(): StubMapping = {
               AuditStub.audit()
               AuthStub.authorised()
               MtdIdLookupStub.ninoFound(nino)
-              DownstreamStub.onError(DownstreamStub.GET, desUri, desStatus, errorBody(desCode))
+              DownstreamStub.onError(DownstreamStub.GET, downstreamUri, downstreamStatus, errorBody(downstreamCode))
             }
 
             val response: WSResponse = await(request(None).get())
             response.status shouldBe expectedStatus
             response.json shouldBe Json.toJson(expectedBody)
           }
+          s"downstream returns an $downstreamCode error and status $downstreamStatus" in new NonTysTest with HasTest
+          s"TYS downstream returns an $downstreamCode error and status $downstreamStatus" in new TysIfsTest with HasTest
```

Comment:
I don't think we actually need line 243 as it doesn't test anything different to 242